pkg/client/client.go

```diff
@@ -325,7 +329,24 @@ func New(c *restclient.Config) (*Client, error) {
 		return nil, err
 	}
 
-	return &Client{client}, nil
+	return &Client{RESTClient: client, config: &config}, nil
+}
+
+// ForGroupVersion returns a copy of the client but without any openshift legacy
+// defaulting.
+// NOTE: This client should be only used in transition period when the legacy API is
+// locked for new types, but the clientsets are not provided.
+func (c *Client) ForGroupVersion(gv *unversioned.GroupVersion) *Client {
+	newConfig := *c.config
+	newConfig.APIPath = "/apis"
+	newConfig.GroupVersion = gv
+
+	newClient, err := restclient.RESTClientFor(&newConfig)
+	if err != nil {
+		glog.Errorf("unable to get client for group %#v", gv)
```

Comment:
print the error? Or even panic? At least, if we avoid adding error return types everywhere.